UserEntries.java

```diff
@@ -84,6 +85,8 @@ private void addPurchase() throws IOException{
 			}
 			System.out.println("You entered: " + amount);
 
+			date.newDate();
```

Comment:
i thought we were deleting the date class to replace with validDate?

Reply:
date is a validDate object

Reply:
the declaration is on line 68